### ISSUE TITLE
Navigation in Documentation

### ISSUE THREAD

**Issue by minoobeyzavi** [minoobeyzavi/Visual-KPI#3]
Most of the Beta Documentations include hyperlinks to more information and some of them are referring to other pages of the documentations. but generally users have to use the back button to access the list of documents and then decide/remember what article to open (whether to move forward or to look back at the previous steps).

It would create a far more intuitive, speedy and satisfactory user experience if two or three small links are available on every page for **Next**, **Previous** and **Main** article pages.

**Comment by rhylton** [comment]
I agree, and I think this is just a feature we need to enable. I will look into it.
